data/core/macros/weapon_specials.cfg

```diff
@@ -94,17 +94,32 @@
     [/petrifies]
 #enddef
 
+#define INTERNAL:COMMON_MARKSMAN
+    id=marksman
+    description= _ "When used offensively, this attack always has at least a 60% chance to hit."
+    value=60
+    cumulative=yes
+    active_on=offense
+#enddef
+
 #define WEAPON_SPECIAL_MARKSMAN
     # Canned definition of the Marksman ability to be included in a
     # [specials] clause.
     [chance_to_hit]
-        id=marksman
-        name= _ "marksman"
-        description= _ "When used offensively, this attack always has at least a 60% chance to hit."
+        {INTERNAL:COMMON_MARKSMAN}
+        name=_"marksman"
         special_note={INTERNAL:SPECIAL_NOTES_MARKSMAN}
-        value=60
-        cumulative=yes
-        active_on=offense
+    [/chance_to_hit]
+#enddef
+
+#define WEAPON_SPECIAL_PRECISE_STRIKE
+    [chance_to_hit]
+        {INTERNAL:COMMON_MARKSMAN}
+        #po: Here, 'precise strike' is used instead of 'marksman' because that term is more suitable for use in melee.
+        name=_"precise strike"
+        #po: This note describes the effects of a special weapon similar to 'marksman'
```

Comment:
@stevecotton Does this work? In terms of the po hint getting processed in multiple lines.

I'm not sure it's necessary to split the comment across two lines here.